invenio_app_ils/records/resolver/loan.py

```diff
@@ -14,7 +14,7 @@
 
 
 def item_resolver(loan_pid):
-    """Resolve an Item given a Loan PID."""
+    """Resolve a Item given a Loan PID."""
```

Comment:
Sorry for being a grammar police but **an** item is correct.

Reply:
👍